pkg/apis/cartographer/v1alpha1/deliverable.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 VMware
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:path=deliverables,categories=all,shortName=dlv
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Source",type="string",JSONPath=`.spec.source['git.url','image']`
+// +kubebuilder:printcolumn:name="Delivery",type="string",JSONPath=".status.deliveryRef.name"
+// +kubebuilder:printcolumn:name="Ready",type="string",JSONPath=`.status.conditions[?(@.type=='Ready')].status`
```

Comment:
you could remove the kubebuilder tags and the description of fields from the file. This is a copy and not the source of CRDs location so you dont need make it descriptive

Reply:
Aren't those required to generate deep copy?

Reply:
CLi uses controller gen to generate fakes. I see that other CRDs have kube builder tags as well. I will create a separate story to clean it up.